app/controllers/transport/border/active/AddAnotherBorderTransportController.scala

```diff
@@ -49,26 +54,29 @@ class AddAnotherBorderTransportController @Inject() (
   def onPageLoad(departureId: String, mode: Mode): Action[AnyContent] = actions.requireData(departureId) {
     implicit request =>
       val viewModel = viewModelProvider(request.userAnswers, departureId, mode)
-      viewModel.count match {
-        case 0 =>
-          Redirect(
-            controllers.transport.border.routes.BorderModeOfTransportController.onPageLoad(departureId, mode)
-          ) //TODO: Change to /add-identification when built
-        case _ => Ok(view(form(viewModel), departureId, viewModel))
-      }
+      Ok(view(form(viewModel), departureId, viewModel))
```

Comment:
So because the `RemoveBorderTransportYesNoController` redirects us back to here we will need to handle the `case 0` separately. For now it might be a TODO pending CTCP-3901